### PR TITLE
TMI2-521 - Removed unnecessary dependencies

### PR DESCRIPTION
**Pull request by kiramarstonTCO**
## Description

[TMI2-521 - Remove unnecessary Apply Admin/Applicant dependencies](https://technologyprogramme.atlassian.net/jira/software/projects/TMI2/boards/331?selectedIssue=TMI2-521)

If the ticket says it shouldn't be there and it is it's because it did need to be there after all 😭 

## Type of change

Please check the relevant options.

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes:

- [ ] Unit Test

- [ ] Integration Test (if applicable)

- [ ] End to End Test (if applicable)

## Screenshots (if appropriate):

Please attach screenshots of the change if it is a UI change:

# Checklist:

- [ ] If I have listed dependencies above, I have ensured that they are present in the target branch.
- [ ] I have performed a self-review of my code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation where applicable.
- [ ] I have ran cypress tests and they all pass locally.

### CODE REVIEW

File: packages/admin/package.json [hunk 1]
```diff
@@ -15,53 +15,30 @@
...
     "gap-web-ui": "*",
     "govuk-frontend": "4.7.0",
-    "gray-matter": "4.0.3",
-    "imap-simple": "^5.1.0",
-    "jsonwebtoken": "^9.0.0",
-    "mailparser": "^3.4.0",
     "moment": "^2.29.4",
-    "moment-timezone": "^0.5.37",
     "next": "12.3.4",
-    "next-logger": "^3.0.1",
```
Comment (jgunnCO) [inline packages/admin/package.json:42]: See comment [here](https://github.com/cabinetoffice/gap-find-web/pull/102#discussion_r1440510526) - could you leave this one in?

Comment (kiramarstonTCO) [reply packages/admin/package.json:42]: Yup, adding back in all the ones that shouldn't have come out now 🥲

File: packages/gap-web-ui/package.json [hunk 2]
```diff
@@ -29,26 +29,16 @@
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-node-resolve": "^13.3.0",
     "@rollup/plugin-typescript": "^8.3.3",
-    "@storybook/addon-actions": "7.4.6",
-    "@storybook/addon-essentials": "7.4.6",
-    "@storybook/addon-interactions": "7.4.6",
-    "@storybook/addon-links": "7.4.6",
     "@storybook/cli": "7.4.6",
-    "@storybook/preset-scss": "1.0.3",
```
Comment (jgunnCO) [inline packages/gap-web-ui/package.json:37]: Can you leave these in as well? If you look at [this file](https://github.com/cabinetoffice/gap-find-apply-web/blob/develop/packages/gap-web-ui/.storybook/main.js) you can see they're all added to the storybook config (and then presumably imported by storybook when it runs).

Comment (kiramarstonTCO) [reply packages/gap-web-ui/package.json]: And they're back!

**Review by iaincooper-tco — APPROVED** [review]

**Pull request merged**